identity-and-access-management/secret-manager/quickstart.mdx

```diff
@@ -0,0 +1,86 @@
+---
+meta:
+  title: Secret Manager - Quickstart
+  description: This page shows you how to get started with Scaleway's Secret Manager.
+content:
+  h1: Quickstart
+  paragraph: This page shows you how to get started with Scaleway's Secret Manager.
+dates:
+  validation: 2023-02-15
+  posted: 2023-02-15
+---
+
+Scaleway’s Secret Manager allows you to conveniently store, access and share sensitive data such as passwords, API keys and certificates. In this quickstart, we show you how to create and manage a secret, how to create and manage a secret's version and how to delete your resources once you are done using them.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [generated an API key](docs/identity-and-access-management/iam/how-to/create-api-keys/) and enabled the `SecretManagerFullAccess` [permission set](/identity-and-access-management/iam/reference-content/permission-sets/)
+</Message>
+
+
+## How to create a secret
+
+1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu. The **Secret Manager** overview page displays.
+2. Click **Create a secret**.
+3. Configure your secret:
+    * Select the region in which you want to create your secret.
+    * Enter a name for your secret.
+    * Optionally, add tags.
+    * Add your secret manually or click <Icon name="plus" /> to import it from a file.
```

Comment:
I wonder if there should be more info about what secrets are meant to look like, because looking at the default secret in the console I see that it's not just a case of entering a password or a key, but seemingly more complex than that?

Reply:
Secrets can be anything form an image to just text, as long as they don't exceed the size limitation